pkg/apis/types.go

```diff
@@ -85,6 +88,19 @@ type RouteConfig struct {
 	Scope uint8 `json:"scope,omitempty"`
 }
 
+// NeighborConfig represents a neighbor (ARP/NDP) entry.
```

Comment:
can you update https://github.com/google/dranet/blob/main/site/content/docs/user/interface-configuration.md ? no need for exhaustive docs, just to reflect this changes, generated ai docs is ok

Reply:
Sure, done!